From 0bc13c72ec0596a0928485d66631c16f1a091af4 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Tue, 10 May 2016 11:23:46 +0800
Subject: [PATCH] arm64: dts: rk3399: disabled the edp on evb2 cros

As the pervious discussed, disabled the edp node for dts. Otherwise
you connected the mipi panel will show white since the edp will force
to use the extend screen.

Change-Id: I3e716f4a9d92d4faae5d7a12d792c71fe16f489e
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-cros.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-cros.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-cros.dts
index b0113d91b729..0a698a404151 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-cros.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-cros.dts
@@ -72,7 +72,7 @@
 };
 
 &edp {
-	status = "okay";
+	status = "disabled";
 
 	ports {
 		edp_out: port@1 {
-- 
2.35.3

